data/moves/animation_pointers.asm

```diff
@@ -3,22 +3,22 @@ BattleAnimations::
 	indirect_table 2, 0
 	indirect_entries GEN1_MOVES, BattleAnimationsGen1
 	indirect_entries GEN2_MOVES, BattleAnimationsGen2
+	indirect_entries GEN3_MOVES, BattleAnimationsGen3
 	indirect_entries $ffff - NUM_BATTLE_ANIMS
 	indirect_entries $ffff, BattleAnimationsNegatives
 	indirect_table_end
 
 
 SECTION "Battle Animations Gen 1", ROMX
-
 INCLUDE "data/moves/animations_gen1.asm"
 
-
 SECTION "Battle Animations Gen 2", ROMX
-
 INCLUDE "data/moves/animations_gen2.asm"
 
-SECTION "Battle Animations Gen Negatives", ROMX
+SECTION "Battle Animations Gen 3", ROMX
+INCLUDE "data/moves/animations_gen3.asm"
 
+SECTION "Battle Animations Gen Negatives", ROMX
 INCLUDE "data/moves/animations_negatives.asm"
```

Comment:
```suggestion

SECTION "Battle Animations Gen 3", ROMX

INCLUDE "data/moves/animations_gen3.asm"


SECTION "Battle Animations Gen Negatives", ROMX

INCLUDE "data/moves/animations_negatives.asm"
```

Reply:
It has been done. Wasn't sure on the formatting is all.